replicatedhq/ship
# set browser no-open in container

What I Did
------------

Prevent the "open browser to continue?" prompt when in a container, just
print the URL to the console.

How I Did it
------------

Always set `NO_OPEN=1` in ship container build.

How to verify it
------------

Build and run the production container

```
curl -sL https://git.io/goreleaser | \
  bash -s -- --snapshot --rm-dist --config deploy/.goreleaser.yml && \
  docker run replicated/ship:latest init stable/mysql
```

Ensure no prompt, and only

```
Please visit the folowing url in your browser:

    http://localhost:8800
```

is displayed.

Description for the Changelog
------------

Prevent ship from displaying an "open browser to continue?" prompt when in a container, instead just print the URL to the console.

![](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2a/Shipwreck_Island.jpg/120px-Shipwreck_Island.jpg)
